### ISSUE TITLE
"Covid" heading in patient results may be too vague

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#3987]
## Area or Component
Patient Experience - results

## Quick Summary
Each of the bolded items in the patient result view is a heading, meaning it's supposed to describe the content underneath. Most of these are ok, but there's a heading labelled "COVID" for the test result that could be better labeled as "test result - COVID" or similar.

## Accessibility Violation
The heading does not describe the topic or purpose of the content that follows it. https://cdc-devtools.dequecloud.com/rule-help/en/heading-not-descriptive

## Action Requested
Add the following headers (depending on whether it's multiplex or not): 

- COVID-19 result
- Flu A result
- Flu B result

## Screenshot
<img width="583" alt="Screen Shot 2022-07-14 at 6 40 43 PM" src="https://user-images.githubusercontent.com/80282552/179129737-caba8310-0f48-4f5b-bd0e-4f49866d980b.png">

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/c3e6b2dc330021e425e01657eaee613c51f9e11c/frontend/src/patientApp/timeOfTest/TestResult.tsx#L1-L1

## Original Deque Issue
heading-not-descriptive | The heading does not describe the topic or purpose of the content that follows it. | "COVID" heading is too vague | Heading does not describe content | https://cdc-devtools.dequecloud.com/rule-help/en/heading-not-descriptive | moderate | Yes | No | Structure | ["body > div#root:nth-of-type(1) > div.App:nth-of-type(1) > div#main-wrapper:nth-of-type(1) > div.pxp-test-results:nth-of-type(2) > main.patient-app.padding-top-105.padding-bottom-4.bg-base-lightest:nth-of-type(1) > div.grid-container.maxw-tablet:nth-of-type(1) > div.card.usa-prose:nth-of-type(1) > div.grid-row:nth-of-type(2) > div.grid-col.usa-prose:nth-of-type(1) > h2.font-heading-sm:nth-of-type(1)"] | The heading does not describe the topic or purpose of the content that follows it. | <h2 class="font-heading-sm">COVID-19</h2> | wcag2aa,wcag246a | rnv7@cdc.gov | Patient Results - Covid | https://dev.simplereport.gov/app/pxp/test-result?plid=2d388b8a-ce92-43e9-bc35-75306ef6df08 |   | 2022-07-14T22:33:30.486Z |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

**Comment by kenieh** [comment]
@lauraykerr @nickisblake this seem like a fairly easy ticket that we can work off of async. Do y'all have any ideas or preferences for what we should rename the title? Some that come to mind:

"COVID-19 test result"
"Test result - COVID-19" (as Emma mentioned in the ticket description)

I lean towards the former. I think it would read better on a screen reader too.

**Comment by nickisblake** [comment]
@lauraykerr @kenieh I also think "Test result - COVID-19" would read better on a screen reader, it's straight to the point.

**Comment by lauraykerr** [comment]
Definitely down to stay concise, but thinking `COVID-19 result` might be descriptive enough (that's what's used on the Test details popup on the Test results page). What do you think @kenieh @nickisblake?

@emmastephenson Assuming this is an issue on multiplex results too, is that correct? In that case, it would be `Flu A result` and `Flu B result`.

![Screen Shot 2022-08-02 at 6 23 26 AM](https://user-images.githubusercontent.com/22102190/182425305-35ba6a5a-ab26-4478-b0a9-fcfe3501361a.png)

**Comment by kenieh** [comment]
@lauraykerr nice catch! I like going for more concise and agree with "COVID-19 result" being the path forward.

**Comment by lauraykerr** [comment]
Great! Added `COVID-19 result`, `Flu A result`, and` Flu B result` to the ticket description and throwing back in engineering backlog for engineering work.

**Comment by lauraykerr** [comment]
@zdeveloper Putting this in eng backlog for when Better Data tackles patient experience.
